src/test/java/seedu/address/logic/parser/FindCommandParserTest.java

```diff
@@ -1,34 +1,55 @@
 package seedu.address.logic.parser;
 
-import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
-import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
 import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
 
 import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
 
 import org.junit.jupiter.api.Test;
 
 import seedu.address.logic.commands.FindCommand;
-import seedu.address.model.person.NameContainsKeywordsPredicate;
+import seedu.address.model.person.predicate.PersonContainsKeywordsPredicate;
 
 public class FindCommandParserTest {
 
     private FindCommandParser parser = new FindCommandParser();
 
     @Test
-    public void parse_emptyArg_throwsParseException() {
-        assertParseFailure(parser, "     ", String.format(MESSAGE_INVALID_COMMAND_FORMAT, FindCommand.MESSAGE_USAGE));
+    public void parse_emptyArg_returnsFindCommand() {
+        List<String> emptyKeywords = Collections.emptyList();
+        FindCommand command = new FindCommand(new PersonContainsKeywordsPredicate(
+                emptyKeywords,
+                emptyKeywords,
+                emptyKeywords,
+                emptyKeywords,
+                emptyKeywords,
+                emptyKeywords,
+                emptyKeywords));
+        assertParseSuccess(parser, "     ", command);
     }
 
     @Test
     public void parse_validArgs_returnsFindCommand() {
+        List<String> emptyKeywords = Collections.emptyList();
+        List<String> nameKeywords = Arrays.asList("Alice", "Bob");
+
         // no leading and trailing whitespaces
         FindCommand expectedFindCommand =
-                new FindCommand(new NameContainsKeywordsPredicate(Arrays.asList("Alice", "Bob")));
-        assertParseSuccess(parser, "Alice Bob", expectedFindCommand);
+                new FindCommand(new PersonContainsKeywordsPredicate(
+                        nameKeywords,
+                        emptyKeywords,
+                        emptyKeywords,
+                        emptyKeywords,
+                        emptyKeywords,
+                        emptyKeywords,
+                        emptyKeywords));
+
+        // space in front of n/ is needed, see javadoc for ArgumentTokenizer.findPrefixPosition
+        assertParseSuccess(parser, " n/Alice Bob", expectedFindCommand);
 
         // multiple whitespaces between keywords
-        assertParseSuccess(parser, " \n Alice \n \t Bob  \t", expectedFindCommand);
+        assertParseSuccess(parser, " n/\n Alice \n \t Bob  \t", expectedFindCommand);
```

Comment:
Just checking, are the `\n`, `\t` etc. format of the flags in this test case are intentional?